### PR TITLE
Add config to enable group creation

### PR DESCRIPTION
**Pull request by erikjohnston**

### CODE REVIEW

File: synapse/config/homeserver.py [hunk 1]
```diff
@@ -35,6 +35,7 @@
 from .workers import WorkerConfig
 from .push import PushConfig
 from .spam_checker import SpamCheckerConfig
+from .groups import GroupsConfig
```
Comment (richvdh) [inline synapse/config/homeserver.py]: no groups.py :(

Comment (erikjohnston) [reply synapse/config/homeserver.py]: That would explain why jenkins is unhappy......

**Comment by richvdh** [comment]
jenkins is still sad tho

**Comment by richvdh** [comment]
needs pep8

**Pull request merged**